python_modules/dagster/dagster/_cli/asset.py

```diff
@@ -3,9 +3,11 @@
 import click
 
 import dagster._check as check
+from dagster._cli.job import get_config_from_args
```

Comment:
this one should be moved into `dagster._cli.workspace.cli_target`, I believe

Reply:
This has been addressed here - https://github.com/jimmyshah/dagster/pull/1/commits/06880bb7c94411a7c4b433db80ee8b45d90b4ba3